### DISCUSSION TITLE
AIK Extraction failed error

### DISCUSSION THREAD

**Discussion by nullptr-t-oss** [SebaUbuntu/TWRP-device-tree-generator#164] [Q&A] [answered]
I'm using Oppo f15(cph2001) A11. Ive tried using twrpdtgen but it says AIK Extraction failed return code 1 . Any fix ??

Ive tried boot image (magisk patched) and recovery image but none of them seems to be working.

![Screenshot_2023-08-20-21-07-08-62_84d3000e3f4017145260f7618db1d683.jpg](https://github.com/twrpdtgen/twrpdtgen/assets/127822226/911b52fe-2ab8-4533-9fb1-81628102ebdf)

**Comment by iibm1** [comment, marked as answer]
try git clone  https://github.com/SebaUbuntu/AIK-Linux-mirror
./AIK-Linux-mirror/unpackimg.sh ./recovery.img
for me : yum install file solved same issue

**Reply by kodeaqua** [reply]
> try git clone https://github.com/SebaUbuntu/AIK-Linux-mirror ./AIK-Linux-mirror/unpackimg.sh ./recovery.img for me : yum install file solved same issue

hi, thanks! the "sudo apt install file" solved my issue too (on Debian WSL2, while on Ubuntu works fine)
